client/my-sites/customer-home/main.jsx

```diff
@@ -368,8 +369,12 @@ class Home extends Component {
 							<WpcomChecklist displayMode={ checklistMode } />
 						</>
 					) : (
-						<Card className="customer-home__card-boxes">
-							<CardHeading>{ translate( 'Site Tools' ) }</CardHeading>
+						<FoldableCard
+							className="customer-home__card-boxes card-heading-21"
+							header={ translate( 'Site Tools' ) }
+							expanded={ ! isMobile() }
+							actionButton={ isMobile() ? null : ' ' }
```

Comment:
I don't understand how this works; why does the value need to be an empty string rather than null if not `isMobile()`?

Reply:
`null` causes the action button not to be overriden, so the usual arrow Gridicon is displayed. In contrast, the empty string causes the Gridicon to be replaced with nothing, preventing the card from being toggled on desktop devices. :)

Reply:
While this is definitely a solution, I'm wondering about the component being conditional on `is_mobile()` (eg. if mobile, display FoldableCard, otherwise, display Card)? It would be more verbose, but magical whitespace values are a little harder to reason about when reading the code. 🤔

Reply:
Interesting, this is non-blocking for me provided that we follow up with a janitorial that chooses between a Card and Foldable Card. It'd also be a chance to move the quick links into its own sub-folder since this main file is getting a bit lengthy.

Reply:
For folks reading dropping the prop here, causes the chevron to be displayed in Desktop screens.

Reply:
I can take care of this later today in this PR. :)

Reply:
I opened #39108 for this in case it's still the preferred approach. :)